config/splat.pspeu.stare.yaml

```diff
@@ -143,11 +143,11 @@ segments:
       - [0x60AF8, .bss, are/e_lock_camera]
       - [0x60B00, .bss, are/e_boss_doors]
       - [0x60B08, .bss, are/create_entity]
-      - [0x60B28, bss, e_collect]
+      - [0x60B28, .bss, are/st_update]
```

Comment:
`st_update` appears earlier in every overlay. It seems likely that it "owns" the BSS here, since ARE has other BSS in between this `g_ItemIconSlots` and the rest of the `e_collect` BSS.

I've moved the declaration to `st_update` and fixed up a few of the splat files accordingly